quickwit/quickwit-opentelemetry/src/otlp/logs.rs

```diff
@@ -446,16 +443,27 @@ fn parse_otlp_logs(
                 .unwrap_or(0);
 
             for log_record in scope_log.log_records {
-                if log_record.time_unix_nano == 0 {
-                    rate_limited_error!(limit_per_min = 10, "skipping record");
-                    num_parse_errors += 1;
-                    continue;
-                }
-                let observed_timestamp_nanos = if log_record.observed_time_unix_nano != 0 {
-                    Some(log_record.observed_time_unix_nano)
+                let observed_timestamp_nanos = if log_record.observed_time_unix_nano == 0 {
```

Comment:
I understand what you meant (great comment by the way). But I don't want to fill a field that we don't really use on quickwit side. It will just take more space.

I would just set `timestamp_nanos` to `observed_timestamp_nanos` if there is one. I consider the user responsible for setting at least one timestamp info. If there is nothing in the record, then we should reject it so the user realizes he's doing bad things.

Reply:
1. I think it should never happen, sending OTEL without ObservedTimestamp seems like be a pretty big disrespect for the standard :smile: 
2. If it did happen, it would be hard for the user to fix the exporter side as the issue would likely come from some tooling they don't really control. I really think that preventing the user from using Quickwit in that (unlikely) case is a bad idea.

Reply:
Let's keep what you did then.